client/changelist/changelist.go

```diff
@@ -21,6 +21,24 @@ func (cl *memChangelist) Add(c Change) error {
 	return nil
 }
 
+// Remove deletes the changes found at the given indices
+func (cl *memChangelist) Remove(idxs []int) error {
```

Comment:
Can we add a test for this method?  I think the `cmd` functions only use file changelists which is why it isn't covered